src/main/webapp/file.xhtml

```diff
@@ -18,8 +18,9 @@
             <ui:param name="pageTitle" value="#{FilePage.fileMetadata.dataFile.displayName} - #{FilePage.file.owner.owner.displayName}"/>
             <ui:param name="dataverse" value="#{FilePage.file.owner.owner}"/>
             <ui:param name="showMessagePanel" value="#{true}"/>
+            <!-- NOTE removed fileAccessRequest logic in order to always display the file access btn -->
             <ui:param name="showAccessFileButtonGroup" value="#{fileDownloadHelper.canDownloadFile(FilePage.fileMetadata)
-                                                                  or (!fileDownloadHelper.canDownloadFile(FilePage.fileMetadata) and FilePage.fileMetadata.dataFile.owner.fileAccessRequest)
```

Comment:
another spot for some rendering clean up

Reply:
Fixed in 7903a8f5d.